src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -464,6 +466,12 @@ const AuthoritiesSearch = ({
     }
   }, [doExport]);
 
+  const onGenerateCQLQueryReport = useCallback(async () => {
+    const fileName = `SearchAuthorityCQLQuery${dayjs().format()}.cql`;
+
+    saveAs(new Blob([query], { type: 'text/plain;charset=utf-8;' }), fileName);
+  }, [query]);
+
```

Comment:
Oooh, this looks very ... simple! @JohnC-80, do you think we should look at refactoring [ExportCsv](https://github.com/folio-org/stripes-components/tree/master/lib/ExportCsv)?

Reply:
@zburke The only downside is that the dependency hasn't published a new version in 4 years... but it has no dependencies itself... The only flag in the codebase is usage of deprecated 'initBlahEvent'... which really, may never actually go away....